test/cpp/util/channelz_sampler.cc

```diff
@@ -0,0 +1,512 @@
+    all_servers.clear();
+    all_sockets.clear();
+    top_channels.clear();
+    id_set.clear();
+  }
+
+  // Store a channel in Json
+  void StoreChannelInJson(const grpc::channelz::v1::Channel& channel) {
+    std::string id = grpc::to_string(GetChannelID(channel));
+    std::string type = "Channel";
+    std::string description;
+    ::google::protobuf::TextFormat::PrintToString(channel.data(), &description);
+    StoreEntityInJson(id, type, description);
+  }
+
+  // Store a subchannel in Json
+  void StoreSubchannelInJson(const grpc::channelz::v1::Subchannel& subchannel) {
+    std::string id = grpc::to_string(GetSubchannelID(subchannel));
+    std::string type = "Subchannel";
+    std::string description;
+    ::google::protobuf::TextFormat::PrintToString(subchannel.data(),
+                                                  &description);
+    StoreEntityInJson(id, type, description);
+  }
+
+  // Store a server in Json
+  void StoreServerInJson(const grpc::channelz::v1::Server& server) {
+    std::string id = grpc::to_string(GetServerID(server));
+    std::string type = "Server";
+    std::string description;
+    ::google::protobuf::TextFormat::PrintToString(server.data(), &description);
+    StoreEntityInJson(id, type, description);
+  }
+
+  // Store a socket in Json
+  void StoreSocketInJson(const grpc::channelz::v1::Socket& socket) {
+    std::string id = grpc::to_string(GetSocketID(socket));
+    std::string type = "Socket";
+    std::string description;
+    ::google::protobuf::TextFormat::PrintToString(socket.data(), &description);
+    StoreEntityInJson(id, type, description);
+  }
+
+  // Store an entity in Json
+  void StoreEntityInJson(std::string& id, std::string& type,
+                         std::string& description) {
+    std::string start, finish;
+    time_t ago = now - int64_t(FLAGS_sampling_interval_seconds);
+    std::stringstream ss;
+    ss << std::put_time(std::localtime(&now), "%F %T");
+    finish = ss.str();  // example: "2019-02-01 12:12:18"
+    ss.str("");
+    ss << std::put_time(std::localtime(&ago), "%F %T");
+    start = ss.str();
+
+    grpc_core::Json obj =
+        grpc_core::Json::Object{{"Task", absl::StrFormat("%s_ID%s", type, id)},
+                                {"Start", start},
+                                {"Finish", finish},
+                                {"ID", id},
+                                {"Type", type},
+                                {"Description", description}};
+    json.mutable_array()->push_back(obj);
+  }
+
+  // Dump data in json
+  std::string DumpJson() { return json.Dump(); }
+
+  // Check if one entity has been recorded
+  bool CheckID(int64_t id) {
+    if (id_set.count(id) == 0) {
+      id_set.insert(id);
+      return true;
+    } else {
+      return false;
+    }
+  }
+
+  time_t now;
+  // private:
+  std::unique_ptr<grpc::channelz::v1::Channelz::Stub> channelz_stub;
+  std::vector<grpc::channelz::v1::Channel> top_channels;
```

Comment:
please suffice all class member fields with `_` (https://google.github.io/styleguide/cppguide.html#Variable_Comments)

Reply:
Done. Thanks for pointing out.